src/components/calltree/CallTree.js

```diff
@@ -63,6 +60,7 @@ class CallTreeComponent extends PureComponent<Props> {
   _appendageColumn: Column;
   _appendageButtons: string[];
   _treeView: TreeView | null;
+  _takeTreeViewRef = treeView => (this._treeView = treeView);
 
   constructor(props: Props) {
```

Comment:
For example here the whole constructor could be done using class properties instead, but this isn't the goal of the patch so I left it out for a later patch.